src/arduino/comms/packet.h

```diff
@@ -0,0 +1,13 @@
+class Packet
```

Comment:
We do still need some I2C Packet protocol for futureproofing so while packager needs overhaul prob not this class

Reply:
Generalized the naming here a bit. Not much else to do here